packages/styles/src/mixins/list/_list-message-view.scss

```diff
@@ -2,6 +2,7 @@
 
 .#{$block} {
   &--message-view {
+    --faListNavigationIndicatorFontSize: 0.75rem;
```

Comment:
`--fa`? Also there's a variable, not sure how important it is to use but `--sapFontSmallSize`